cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Cloud/Infra VMs."""
+import pytest
+
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm_views import PolicySimulationDetailsView
+from cfme.common.vm_views import PolicySimulationView
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers', 'uses_cloud_providers', 'setup_provider'),
+    pytest.mark.tier(2),
+    pytest.mark.provider(
+        classes=[CloudProvider, InfraProvider],
+        selector=ONE_PER_CATEGORY,
+        scope="module"
+    ),
+]
+
+
+@pytest.fixture
+def vm_and_filter(appliance, provider):
+    collection = provider.appliance.provider_based_collection(provider)
```

Comment:
@john-dupuy 
Just a question:
- Do you really need this as a fixture?
Because this fixture does not create or teardown anything. So we can add this to test case itself and fetch this data.

Reply:
@ganeshhubale Right now, this doesn't really need to be a fixture. However this is the first test for the policy simulation page, and it is likely that future tests will make use of this fixture.